gr-uhd/lib/usrp_source_impl.cc

```diff
@@ -153,7 +124,7 @@ namespace gr {
     usrp_source_impl::_set_center_freq_from_internals(size_t chan, pmt::pmt_t direction)
     {
       _chans_to_tune.reset(chan);
-      if (pmt::eqv(direction, ant_direction_tx())) {
+      if (pmt::eqv(direction, pmt::mp("TX"))) {
```

Comment:
since there are potentially many tags, this (ad hoc `mp`) can get a bit hairy here.